packages/vaadin-virtual-list/test/wheel-scrolling.test.js

```diff
@@ -1,7 +1,7 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
 import Sinon from 'sinon';
-import { Virtualizer } from '..';
+import { Virtualizer } from '../virtualizer';
```

Comment:
Let's add `.js` extension for consistency.

Reply:
Done